### PR TITLE
LOC CHECKIN | dotnet/roslyn master | 20200116

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

File: src/CodeStyle/CSharp/Analyzers/xlf/CSharpCodeStyleResources.zh-Hans.xlf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en" target-language="zh-Hans" original="../CSharpCodeStyleResources.resx">
+  <file datatype="xml" source-language="en" target-language="zh-HANS" original="../CSharpCodeStyleResources.resx">
```
Comment (dibarbet) [inline src/CodeStyle/CSharp/Analyzers/xlf/CSharpCodeStyleResources.zh-Hans.xlf]: is the capitalization change necessary?

Comment (janbaltus) [reply src/CodeStyle/CSharp/Analyzers/xlf/CSharpCodeStyleResources.zh-Hans.xlf]: We fixed the language core caitalization,

**Comment by dibarbet** [comment]
@v-zbsail @janbaltus looks like there are some merge conflicts in the translations, could you check?

**Comment by v-zbsail** [comment]
@dibarbet Those two strings in conflict were changed in master branch after we created this PR (https://github.com/dotnet/roslyn/pull/40789)
Feel free to keep the changes from master. We will update the translations in the next cycle.

@janbaltus FYI

**Comment by dibarbet** [comment]
Thanks.  Kept the changes from master.

**Pull request merged**